src/memory-analyzer/analyze_symbol.cpp

```diff
@@ -191,6 +193,43 @@ exprt symbol_analyzert::get_char_pointer_value(
   }
 }
 
+exprt symbol_analyzert::get_pointer_to_member_value(
+  const exprt &expr,
+  const pointer_valuet &pointer_value,
+  const source_locationt &location)
+{
+  PRECONDITION(expr.type().id() == ID_pointer);
+  PRECONDITION(!is_c_char(expr.type().subtype()));
+  std::string memory_location = pointer_value.address;
+  PRECONDITION(memory_location != "0x0");
+  PRECONDITION(!pointer_value.pointee.empty());
+
+  std::string struct_name;
+  std::string member_offset_string;
+  split_string(
+    pointer_value.pointee, '+', struct_name, member_offset_string, true);
+  size_t member_offset = safe_string2size_t(member_offset_string);
+
+  const symbolt *struct_symbol = symbol_table.lookup(struct_name);
+  DATA_INVARIANT(struct_symbol != nullptr, "unknown struct");
+  const auto &struct_expr = struct_symbol->symbol_expr();
+  const struct_tag_typet &struct_tag_type =
+    to_struct_tag_type(struct_expr.type());
+  const struct_typet struct_type = ns.follow_tag(struct_tag_type);
+
+  size_t current_offset = 0;
+  for(size_t i = 0; i < struct_type.components().size(); ++i)
+  {
+    const struct_typet::componentt &component = struct_type.components()[i];
+    const auto &bv_type = to_bitvector_type(component.type());
```

Comment:
Obviously won't work if there is a non-bitvector member. But at least it will fail gracefully.